openapi/openapi.json

```diff
@@ -103,11 +111,28 @@
                                             "attributes": {
                                                 "type": "object",
                                                 "additionalProperties": false,
+                                                "required": ["templateName", "owner"],
                                                 "properties": {
                                                     "templateName": {
                                                         "title": "Questionnaire template name",
                                                         "type": "string",
                                                         "pattern": "^[a-z]{1,20}(?:-[a-z]{1,20}){0,10}$"
+                                                    },
+                                                    "owner": {
+                                                        "type": "object",
+                                                        "additionalProperties": false,
+                                                        "required": ["id", "isAuthenticated"],
+                                                        "properties": {
+                                                            "id": {
+                                                                "title": "Owner Id",
+                                                                "type": "string",
+                                                                "format": "uri",
+                                                                "pattern": "^urn:(uuid|fdc:gov.uk:2022):"
```

Comment:
have a max length on this regex to stop huge strings. Whole string length should have a max of 64, I think!